src/components/layouts/PublicPageLayout/PublicPageLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Grid, WithStyles, withStyles } from '@material-ui/core';
+
+import styles from './styles';
+
+export interface PublicPageLayoutProps extends WithStyles<typeof styles> {
+  children: JSX.Element;
+}
+
+function PublicPageLayout(props: PublicPageLayoutProps): JSX.Element {
+  const { children, classes } = props;
+  return (
+    <Grid
+      className={classes.root}
+      container
+      direction='row'
+      justify='center'
+      alignItems='center'
+    >
+      <Grid item className={classes.children}>
+        {children}
```

Comment:
Also, is this grid item going to hold all of children, even when children can have multiple components? I thought grid item is usually used for only one component.

Reply:
it'll just be one thing

Reply:
Sounds good to me : )